docs/charts.md

```diff
@@ -45,6 +45,7 @@ The `Chart.yaml` file is required for a chart. It contains the following fields:
 ```yaml
 name: The name of the chart (required)
 version: A SemVer 2 version (required)
+kubeVersion: A SemVer range of compatible Kubernetes SemVer versions (optional)
```

Comment:
Sorry for being pedantic, could we remove the second SemVer as it seem redundant.

Reply:
No worries. I just wanted to be extra specific